CONTRIBUTORS.md

```diff
@@ -160,3 +163,5 @@ See [CONTRIBUTING.md](CONTRIBUTING.md) for details on how to contribute to Cesiu
 * [Cody Guldner](https://github.com/burn123)
 * [Florent Cayré](https://github.com/fcayre)
 * [Nacho Carnicero](https://github.com/nacho-carnicero)
+* [Yonatan Kra](https://github.com/yonatankra)
```

Comment:
Did you mean to add yourself to this section as well as the **webiks** section above?